README.md

```diff
@@ -36,6 +36,7 @@ Make sure to define `COVERITY_SCAN_TOKEN` in your
 Here's a non-exhaustive list of projects using this action.
 
 * [Breakpad](https://github.com/google/breakpad/blob/HEAD/.github/workflows/coverity.yml) (configure+make)
+* [libexpat](https://github.com/libexpat/libexpat/blob/master/.github/workflows/coverity-scan.yml) (cmake+make)
```

Comment:
s/master/HEAD/

Reply:
@vapier can do, coming up. To be robust towards rename of the master branch? What the idea?

Reply:
@vapier PS: Pushed

Reply:
yes, if the main branch is renamed, I don't have to track it